tests/jax/latest/wmt.libsonnet

```diff
@@ -33,10 +38,17 @@ local tpus = import 'templates/tpus.libsonnet';
   },
   local wmt = common.runFlaxLatest {
     modelName:: 'wmt',
-    extraDeps+:: ['tensorflow_text sentencepiece'],
+    extraDeps+:: ['tensorflow_text sentencepiece', 'protobuf==3.20.*', 'tensorflow_datasets'],
```

Comment:
Minor: seems like two formatting styles are being mixed here. tensorflow_text and sentencepiece are separated by space and the others are separated by commas

Reply:
I agree, I plan to remove the dependencies that has been added here soon after the bug is fixed.